README.md

```diff
@@ -275,6 +275,9 @@ ENV vars can control the defaults:
 - `CONFIG_OUTPUT_DIR` - To override the output path where configs are dumped (default: `/tmp/config`)
 - `VAULT_CONFIG_ROOT` - Vault root path where config envs are stored (default: `secret/aenode/config`)
 - `VAULT_CONFIG_FIELD` - Name of the field where the configuration YAML is stored (default: `node_config`)
+- `CONFIG_ENV` - Vault configuration env (default: `$(DEPLOY_ENV)`)
+- `LIMIT` - Ansible --limit option (default: `tag_env_$(DEPLOY_ENV):&tag_role_aenode`)
+- `PYTHON` - Full path of the python interpreter (default: `/usr/bin/python3`)
```

Comment:
add 
- `ANSIBLE_EXTRA_PARAMS` - Additional params to pass to `ansible-playbook` (i.e `-i {host},`)